<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TonUINOUploader</class>
 <widget class="QDialog" name="TonUINOUploader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TonUINO Uploader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QLineEdit" name="localFileLineEdit">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="startPushButton">
       <property name="text">
        <string>Start Upload</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="refreshPortPushButton">
       <property name="text">
        <string>Refresh Ports</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="hwVariantLabel">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>HW Variante</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="downloadSD">
       <property name="text">
        <string>Download SD Karte</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="hwTypeCheckBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="hwVariantCheckBox"/>
     </item>
     <item row="3" column="6">
      <widget class="QCheckBox" name="timestampCheckBox">
       <property name="text">
        <string>Timestamps</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Hardware Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="consolePushButton">
       <property name="text">
        <string>Console Log</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="portCheckBox"/>
     </item>
     <item row="1" column="3">
      <widget class="QToolButton" name="localFileToolButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" rowspan="2" colspan="4">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TonUINO-TNG Uploader</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" alignment="Qt::AlignBottom">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Lokales Firmware File:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="console">
     <property name="font">
      <font>
       <family>Courier New</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>console</tabstop>
  <tabstop>hwTypeCheckBox</tabstop>
  <tabstop>hwVariantCheckBox</tabstop>
  <tabstop>portCheckBox</tabstop>
  <tabstop>refreshPortPushButton</tabstop>
  <tabstop>localFileLineEdit</tabstop>
  <tabstop>localFileToolButton</tabstop>
  <tabstop>startPushButton</tabstop>
  <tabstop>downloadSD</tabstop>
  <tabstop>consolePushButton</tabstop>
  <tabstop>timestampCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
